challenges/towers-of-hanoi/towers_of_hanoi.py

```diff
@@ -2,12 +2,15 @@ def towers_of_hanoi(n):
     """Validate tower argument and trigger recursion."""
     if type(n) is not int:
         raise TypeError('Please enter an integer!')
-    return(move_tower(n, 'A', 'C', 'B'))
+    moves = move_tower(n, 'A', 'C', 'B')
+    return moves
 
 
 def move_tower(discs, origin, target, helper):
     """Handle recursive shuffle of discs."""
+    temp = []
```

Comment:
This resets the list of moves each time the function is called. If you want to keep a running total of the moves, you should set the list of moves to an empty list in the base `towers_of_hanoi()` function. Then you can either pass the list of moves in as an additional parameter, or you can define the recursing `move_tower()` function inside the `towers_of_hanoi()` and thereby have access to it to append to.